docs/reference/contactproperties.md

```diff
@@ -53,13 +54,24 @@ With three values, the first solid (corresponding to `material1`) uses asymmetri
 Finally, with four values, both solids use asymmetric coefficients, first two for the first solid and last two for the second solid.
 The two friction directions are defined for each faces of the geometric primitives and match with the U and V components used in the texture mapping.
 Only the `Box`, `Plane` and `Cylinder` primitives support asymmetric friction.
-If another primitive is used, only the first value will be used for symetric friction.
+If another primitive is used, only the first value will be used for symmetric friction.
 WEBOTS\_HOME/projects/sample/howto/asymmetric\_friction/worlds/asymmetric\_friction1.wbt contains an example of fully asymmetric friction.
 
 - The `frictionRotation` allows the user to rotate the friction directions used in case of asymmetric `coulombFriction` and/or asymmetric `forceDependentSlip`.
 By default, the directions are the same than the ones used for texture mapping (this can ease defining an asymmetric friction for a textured surface matching the rotation field of the corresponding TextureTransform node).
 WEBOTS\_HOME/projects/sample/howto/asymmetric\_friction/worlds/asymmetric\_friction2.wbt illustrates the use of this field.
 
+- The `rollingFriction` field specifies the coefficients of rolling/spinning friction.
+The field can have up to three coefficients, using ODE's nomenclature they are [rho, rho2, rhoN].
+The field accepts only positive values or -1.0, where -1.0 corresponds to infinity.
+For a value of zero no rolling friction is applied.
+`rho` is the rolling friction coefficient in the first friction direction.
+`rho2` is the rolling friction coefficient in the second friction direction, perpendicular to that of `rho`.
+`rhoN` is the rolling friction coefficient around the normal direction.
+
+WEBOTS\_HOME/projects/sample/howto/rolling\_friction/worlds/rolling\_friction.wbt illustrates the effect of the different coefficients.
```

Comment:
A hyperlink to the GitHub file would be welcome here (see other places in the documentation where this is done).